cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-configmaps-secrets-env-var.adoc

```diff
@@ -0,0 +1,86 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-configmaps-secrets-envvar"]
+= Tutorial: ConfigMaps, secrets, and environment variables
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-configmaps-secrets-envvar
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 05-07-2024
+
+This tutorial shows how to configure the OSToy application by using xref:../../nodes/pods/nodes-pods-configmaps.adoc[config maps], link:https://docs.openshift.com/container-platform/4.15/cicd/builds/creating-build-inputs.html#builds-input-secrets-configmaps_creating-build-inputs[secrets], and link:https://docs.openshift.com/container-platform/3.11/dev_guide/environment_variables.html[environment variables]. See these linked topics for more information.
+
+== Configuration using ConfigMaps
+Config maps allow you to decouple configuration artifacts from container image content to keep containerized applications portable.
+
+.Procedure
+* In the OSToy app, click *Config Maps* in the left menu, displaying the contents of the config map available to the OSToy application. The code snippet shows an example of a config map configuration:
++
+.Example output
+[source,text]
+----
+global
+  maxconn 50000
+  log /dev/log local0
+  user haproxy
+  group haproxy
+  stats socket /run/haproxy/admin.sock user haproxy group haproxy mode 660 level admin
+  nbproc 2
+  nbthread 4
+  cpu-map auto:1/1-4 0-3
+  ssl-default-bind-ciphers
+  ssl-default-bind-options ssl-min-ver TLSv1.2 no-tls-tickets
+
+frontend www.mysite.com
+  bind 10.0.0.3:80
+  bind 10.0.0.3:443 ssl crt /etc/ssl/certs/mysite.pem
+  http-request redirect scheme https unless { ssl_fc }
+  use_backend api_servers if { path_beg /api/ }
+  default_backend web_servers
+----
```

Comment:
The above is not the configmap shown in the OSToy application but rather is an haproxy configuration.  Was this intended to be here?